.github/workflows/aam-integration-build-and-publish.yml

```diff
@@ -0,0 +1,286 @@
+name: aam-integration-build-and-publish
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/aam-integration-build-and-publish.yml'
+      - 'application/aam-integration/**'
+  push:
+    tags:
+      - 'aam-integration/**'
+    branches:
+      - main
+    paths:
+      - '.github/workflows/aam-integration-build-and-publish.yml'
+      - 'application/aam-integration/**'
+env:
+  REGISTRY_IMAGE: ghcr.io/aam-digital/aam-integration
```

Comment:
We are building this as a separate system, not as a module within the "normal" backend? Are there specific requirements for this, as the reporting API for integration is then part of the other backend, both interacting with the same external platform?